src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -604,11 +604,12 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 Account a = new Account(
                     Phone = c.Phone,
                     Fax = c.Fax,
-                    OwnerId = c.OwnerId
+                    OwnerId = c.OwnerId,
+                    RecordTypeId = UTIL_CustomSettingsFacade.getSettings().Account_Processor__c
```

Comment:
There was no recordtype specified before for this line of code. Since this appeared to me at the time to be the code that is in charge of creating an account when a contact is created, that it should look at the settings to get the default record type. I set it here so that when we pass the account over for naming, I can check to see what naming pattern should be used based on the recordtype.

Reply:
Right. The question is what was the record type being used before you made the modification. Was it the default for the profile or...? I just want to get an understanding on who might be affected by the change. Just in case someone considered this "a feature".

Reply:
I was still using org default but happening further down the method (see line 625). I should probably be clearer and specify the defaultRecTypeId on line 608 which is declared at the top of the class. That said, I think having just noticed line 625, I can probably get rid of lines 625-627

Reply:
All good. Same behavior as before. We will just use the static variable instead.